Lib/test/test_statistics.py

```diff
@@ -2914,10 +2912,8 @@ def setUp(self):
     def tearDown(self):
         sys.modules['statistics'] = statistics
 
-    # TODO: RUSTPYTHON, ValueError: math domain error
-    @unittest.expectedFailure
-    def test_inv_cdf(self): # TODO: RUSTPYTHON, remove when this passes
-        super().test_inv_cdf() # TODO: RUSTPYTHON, remove when this passes
+    def test_inv_cdf(self):
+        super().test_inv_cdf()
```

Comment:
The `# TODO: RUSTPYTHON, remove when this passes` comments mean that this should be removed **completely**.